services/secure-gateway/main.go

```diff
@@ -52,12 +55,29 @@ func NewApp(configuration *config.Configuration, token security.JWTToken) http.H
 	enforcer.EnableLog(false)
 	enforcer.EnableEnforce(configuration.Security.Enforce)
 
-	router := NewRouter(configuration)
-	authenticationMiddleware := security.AuthenticationMiddleware(configuration.Security)
-	authorizationMiddleware := security.AuthorizationMiddleware(enforcer)
+	auditor, err := audit.New(os.Stdout)
+	if err != nil {
+		panic("Unable to create security auditing log")
+	}
+	if configuration.Security.Enforce {
```

Comment:
Consider one function like LogEnforserStatus that recieves configuration and logs the state according to it

Reply:
We'll keep it as is for now, if we encounter problems with it - we'll fix it.